src/shared/localization.ts

```diff
@@ -17,7 +17,10 @@ export const defaultStrings = {
     commands: {
         blockquote: shortcut("Blockquote"),
         bold: shortcut("Bold"),
-        code_block: shortcut("Code block"),
+        code_block: {
```

Comment:
I agree in general, but suggest `snake_case` in the instance of commands since that matches the [basic ProseMirror schema](https://prosemirror.net/docs/ref/#schema-basic).

Reply:
I might actually opening Pandora's box here:
- It looks like we are not consistent in the naming of schema nodes (at least [there is this exception](https://github.com/StackExchange/Stacks-Editor/blob/eb7e1c50204b3884ccefc1390942944845d0b126/src/rich-text/schema.ts#L429) that propagates in the localization file)
- In the [defaultStrings object](https://github.com/StackExchange/Stacks-Editor/blob/eb7e1c50204b3884ccefc1390942944845d0b126/src/shared/localization.ts#L16) we group things around the `commands` keyword although not everything is a command really. All are `menu_entries` though.

I think I will make a follow up PR with a refactor, trying to normalize a bit the conventions.
For this one I am not going to take any action apart from renaming `helpText` to `help_text` to follow the convention used by the majority of the items in the `defaultStrings` object.

Reply:
Ah yeah - if there are any items in there that are `camelCased`, then that's an oversight and should be remediated (follow-up PR). I don't recall the exact reason I made these `snake_case` to begin with, but I feel like it was probably a good reason. I'll think on it a bit and see if I can't remember.